sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -734,14 +794,14 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
       std::string Prefix;
       std::string PreviousContinuationToken;
       int32_t PageSizeHint = int32_t();
-      Models::FilesAndDirectoriesListSinglePage SinglePage;
+      FilesAndDirectoriesListSinglePage SinglePage;
```

Comment:
ditto

Reply:
It has already been flattened in `ListFilesAndDirectoriesSinglePageResult`. This one is not visible to customers.